xml/System.Threading/Interlocked.xml

```diff
@@ -1971,7 +1971,7 @@ If `comparand` and the object in `location1` are equal by reference, then `value
 ## Remarks  
  This method was added to the <xref:System.Threading.Interlocked> class in the [!INCLUDE[net_v45](~/includes/net-v45-md.md)] as a convenience; it's a wrapper for the <xref:System.Threading.Thread.MemoryBarrier%2A?displayProperty=nameWithType> method.  
   
- <xref:System.Threading.Interlocked.MemoryBarrier%2A> is required only on multiprocessor systems that have weak memory ordering (for example, a system that employs multiple Intel Itanium processors).  
+ <xref:System.Threading.Interlocked.MemoryBarrier%2A> is required only on multiprocessor systems that have weak memory ordering (for example, a system that employs an ARM processor).  
```

Comment:
The statement is a bit confusing to me, it seems to suggest that `MemoryBarrier` is not required on x86/x64. Currently a memory barrier may be required for correctness on any supported processor arch, so I would just remove the statement completely.